Basic/DistributedAuthoritySocialHub/Assets/Scripts/UI/IngameUI/UIUtils.cs

```diff
@@ -70,5 +70,16 @@ internal static VisualElement GetFirstChild(this VisualElement element)
             }
             return null;
         }
+
+        /// <summary>
+        /// Authentication UserName gets a # with a number appended at the end. This method returns the player name without the number.
+        /// </summary>
+        /// <param name="authenticationUserName">Name with # appended</param>
+        /// <returns>Name without the last hash and following numbers</returns>
+        public static string ExtractPlayerNameFromAuthUserName(string authenticationUserName)
```

Comment:
Can be internal.

Reply:
there is at least always one public in my PRs 😂